src/objects/page/page.stories.mdx

```diff
@@ -45,3 +48,28 @@ The header block is optional.
     }}
   </Story>
 </Canvas>
+
+## Page with Alert
+
+The Page object also has an optional `alert` block that can be used to pass an [Alert component](/?path=/docs/components-alert--basic).
+
+<Canvas>
+  <Story
+    name="Example with Alert"
+    height="400px"
+    parameters={{
+      layout: 'fullscreen',
+      docs: { source: { code: exampleDemoWithAlertSource } },
+    }}
+  >
+    {() => {
+      useEffect(() => {
+        // Set this story's `body` element to full-height
+        document.body.style.height = '100%';
+        // Prevent Storybook's container from affecting this layout
+        document.querySelector('#root').style.display = 'contents';
+      });
+      return exampleDemoWithAlert();
+    }}
```

Comment:
If this is used on every story, it can probably be moved to a decorator on the `Meta` element. See #1918 for an example.